content/2021/saam-mobility/_index.md

```diff
@@ -141,7 +141,7 @@ The Technical Program Committee is an independent panel of expert volunteers and
 
 	{{< grid/div class="margin-top-30" isMarkdown="false">}}
 	    {{< grid/div class="col-xs-24 col-sm-8 col-sm-offset-2 text-center" isMarkdown="false">}}
-	        {{< bootstrap/button linkClass="btn-primary btn-wide" href="cfp.pdf">}}Call for Paper{{</bootstrap/button>}}
+	        {{< bootstrap/button linkClass="btn-primary btn-wide" href="cfp.pdf">}}Call for Papers{{</bootstrap/button>}}
```

Comment:
I'm noticiing that you are not using the new pdf you added to this patch. Is this intended? Also you are deleting the pdf that is being used here.

Reply:
It looks like we're  using the same filename for the different versions. So, when we updated, Philippe  named the new version cfp.pdf and gave the old file its version name. I checked the downloads and it seems to be working.